app/settings.py

```diff
@@ -2,30 +2,16 @@
 
 import os
 import tomllib
-from datetime import date
 
 from dotenv import load_dotenv
 
 from app.settings_utils import read_bool
 from app.settings_utils import read_list
-from app.settings_utils import support_deprecated_vars
 
 load_dotenv()
 
-APP_HOST = support_deprecated_vars(
-    new_name="APP_HOST",
-    deprecated_name="SERVER_ADDR",
-    until=date(2024, 1, 1),
-)
-APP_PORT = None
-_app_port = support_deprecated_vars(
-    new_name="APP_PORT",
-    deprecated_name="SERVER_PORT",
-    until=date(2024, 1, 1),
-    allow_empty_string=True,
-)
-if _app_port:
-    APP_PORT = int(_app_port)
+APP_HOST = os.environ["APP_HOST"]
```

Comment:
update docker configs as well

Reply:
Should be good I think:

![image](https://i.cmyui.xyz/O2bI5MCmqyQbAF0.png)

We previously deprecated SERVER_HOST and SERVER_PORT back in Sep 2023; this just fully removes support for the already-deprecated vars.

Reply:
https://github.com/osuAkatsuki/bancho.py/blob/4310cf0769f903c3c9d81dc9cb109986530405a1/docker-compose.yml#L89

Reply:
(same thing in test yml)

Reply:
Oo ty didn't notice

Reply:
Aha, in our deprecation msg we typo'd SERVER_ADDR to SERVER_HOST